AndelaOSP/art-dashboard
# #160254384 Asset page re-fetching data once accessed

## What does this PR do?
Fixes bug on asset page invoking api call for data in the store each time it's accessed

## Description of Task to be completed?
### Steps to reproduce
1. Visit `/assets` page 
2. Navigate to a different page
3. Go back to `/assets` page either via the navigation bar or browser back button

### Expected
- To see a populated assets table 

### Actual
- Another API request is made to fetch data, which is already in the store, thereby showing the loader

## How should this be manually tested?
- Once logged in
- Visit assets Lists page 
- Navigate to a different page
- Go back to assets Lists page either via the navigation bar or browser back button
- The data is displayed immediately without loader icon.

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160254384

## Any background context you want to add?
N/a

## Important notes
N/a 

## Packages installed
N/a

## Deployment note
N/a

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
